ui/src/components/schemas/Schemas.tsx

```diff
@@ -16,23 +16,27 @@ export function Schemas() {
     <SiderLayoutContent
       description="Verifiable credential schemas help to ensure the structure and data formatting across different services."
       extra={
-        <Space align="start" size="middle">
-          <Button
-            icon={<IconCreditCardPlus />}
-            onClick={() => navigate(generatePath(ROUTES.issueCredential.path))}
-            type="default"
-          >
-            {ISSUE_CREDENTIAL}
-          </Button>
+        <Row gutter={[8, 8]}>
```

Comment:
Maybe we can add a little bit of space between the paragraph and the buttons?

<img width="529" alt="Captura de pantalla 2023-05-11 a las 20 00 54" src="https://github.com/0xPolygonID/issuer-node/assets/6737805/7e8b3766-b392-4093-b596-bfee6c5624ae">

Reply:
Yes, the problem is not there, but I'll add it